tensorboard/compat/__init__.py

```diff
@@ -44,13 +44,15 @@ def tf():
     ImportError: if a TF-like API is not available.
   """
   try:
-    _importlib.import_module('tensorboard.compat.notf')
+    from tensorboard.compat import notf  # pylint: disable=g-import-not-at-top
```

Comment:
While we’re here, is it intentional that this module is not returned?

Reply:
Yes.  We only care about the presence or absence of the module (controlled by build deps, as explained in the docstring).

Reply:
Great; thanks.